src/alchemlyb/estimators/ti_.py

```diff
@@ -105,3 +105,46 @@ def fit(self, dHdl):
         self.states_ = means.index.values.tolist()
 
         return self
+
+    def separate_dhdl(self):
```

Comment:
possibly separate PR + tests?

Is it only used for the workflow or more general?

Reply:
This is actually used by the plot convergence. I will move this and the plotting to another PR